bom/application/pom.xml

```diff
@@ -3280,7 +3281,11 @@
                 <version>${es-module-shims.version}</version>
                 <scope>runtime</scope>
             </dependency>
-            
+            <dependency>
+                <groupId>org.zeroturnaround</groupId>
```

Comment:
I like zt-exec for sure but why are we needing to add this vs all the process exec utilities we already have?

Reply:
What process exec utilities we have are you referring to? `java.lang.ProcessBuilder` or something specific?